src/components/NavAccordion.tsx

```diff
@@ -0,0 +1,52 @@
+import { Icon } from "@canonical/react-components";
+import { FC, ReactNode } from "react";
+import { useLocation } from "react-router-dom";
+
+export type AccordionNavMenus = "permissions" | "storage";
```

Comment:
```suggestion
export type AccordionNavMenu = "permissions" | "storage";
```
Singular seems better suiting as this type describes one menu at a time, not a set itself.